Revise SRE Agent overview for clarity and consistency (#127839)
Azure/azure-content: articles/sre-agent/overview.md

ms.service: azure-sre-agent
---

> # SRE Agent Overview?

Reviewer: ```suggestion
# Overview of Azure SRE Agent Preview
```

> SRE Agent is designed to automate operational work and reduce toil, enabling developers and operators to focus on high-value tasks. By streamlining repetitive and complex processes, SRE Agent accelerates innovation and improves reliability across cloud and hybrid environments.

Reviewer: ⚠️ We need a different version than what you have here.

What are the elements of this intro that you'd like to make sure to highlight? To be honest, the text here I find confusing and bordering on marketing speak (for example, "accelerates innovation").

If I know more about what you're most interested in expressing, I can help craft something clearer for the docs.

> Operational tasks often involve managing Azure resources or integrating with on-premises and SaaS systems. These tasks can require repetitive actions or complex orchestration across multiple tools. SRE Agent provides the ability to connect to these tools and automate workflows end-to-end.

Reviewer: ```suggestion
Typical operational tasks often include managing multiple Azure resources along with on-prem and SaaS systems. These tasks are often repetitive or require orchestrating together multiple tools to provide the insights you need. SRE Agent lets gives you AI-driven platform to connect to those systems together and automate the workflow end-to-end.
```

## What is SRE Agent?

> SRE Agent is a service that brings automation and intelligence to site reliability engineering practices. It helps organizations reduce manual effort, improve system uptime, and deliver consistent operational outcomes. The agent integrates with Azure services and external systems to execute operational tasks with minimal human intervention.

Reviewer: ```suggestion
SRE Agent is a service that brings automation and intelligence to site reliability engineering practices. It helps you reduce manual effort, improve system uptime, and deliver consistent operational outcomes. As the agent integrates with both Azure services and external systems, it executes operational tasks with minimal human intervention.
```

## Primary use cases

Automate incidents: Connect to incident management platforms to automate triage, mitigation, and resolution, reducing mean time to recovery (MTTR) and improving service availability.

> Automate scheduled workflows: Set up proactive alerting and actions to automate routine and repetitive tasks that run on a defined schedule.

Reviewer: ```suggestion
- **Automate incidents**: Connect to incident management platforms to automate triage, mitigation, and resolution, reducing mean time to recovery (MTTR) and improving service availability.

- **Automate scheduled workflows**: Set up proactive alerting and actions to automate routine and repetitive tasks that run on a defined schedule.
```

Watch the following video to see SRE Agent in action.

<br>

> [!VIDEO https://www.youtube.com/embed/DRWppVNOTqQ?si=FJ9dNk5uY1kUET-R]

> ## How Does It Work?

Reviewer: ```suggestion
## How does SRE Agent work?
```
**Tip**: Titles must be in sentence case.

> SRE Agent combines built-in expertise with full customization capabilities. Out of the box, it understands and manages Azure resources for specific services, providing intelligent defaults for common operational tasks. At the same time, it offers flexibility to incorporate domain-specific knowledge, custom runbooks, and integrations with tools and data sources such as observability and monitoring platforms. This extensibility ensures that SRE Agent can adapt to diverse environments and operational requirements.

Reviewer: ```suggestion
SRE Agent combines fine-tuned Azure expertise with full customization capabilities. Out of the box, SRE Agent understands and manages Azure resources for specific services, providing intelligent defaults for common operational tasks. At the same time, it offers flexibility to incorporate domain-specific knowledge, custom runbooks, and integrations with tools and data sources such as observability and monitoring platforms. This extensibility ensures that SRE Agent can adapt to your environment and operational requirements.
```
A few questions...
- When you say, "built-in expertise", what do you mean? Are you referring to tools and fine-tuned sub-agents? I know we don't want to explain the sub-agents in the docs, but we need to get more specific here.

- What do you mean by "understands Azure resources"? We need to get more specific here as well.

Use the following steps to start working with Azure SRE Agent.

> # [Explore](#tab/explore)

Reviewer: Do you want to remove the tabs from this section all together? Part of your edit removed the "Explore" tab, but left remnants of the tab syntax still in place. We either need content for the "Explore" tab, or I can remove all the tab syntax from the doc.

> # [Handle an incident](#tab/incident)

Reviewer: ```suggestion

```

1. Enable integrations:  

    - Incident management tools: Link to ServiceNow, link to PagerDuty, or use Azure Monitor alerts.
  

>     - Create a new Incident response plan with custom instructions on how incident should be handled

Reviewer: ```suggestion
    - Create a new incident response plan with custom instructions detailing how to handle incidents.
```
